scripts/models/repo.js

```diff
@@ -0,0 +1,12 @@
+// 'use strict';
+
+$.ajax({
+  url: 'https://api.github.com/user/repos?type=owner',
+  method: 'GET',
+  headers: {
```

Comment:
**TODO FOR LOST POINTS**
Good job sending headers here, but the instructions said the header should actually be sent via a proxy. Search through past lab starter code to see how the server file is setup with ``express-request-proxy``.